src/ESPEasyEth.cpp

```diff
@@ -1,9 +1,10 @@
-#include "ESPEasyEth.h"
-
 #ifdef HAS_ETHERNET
 
+#include "ESPEasyEth.h"
+#include "ESPEasyNetwork.h"
 #include "ETH.h"
 #include "ESPEasy-Globals.h"
+#include "eth_phy/phy.h"
 
 bool ethUseStaticIP() {
   return Settings.ETH_IP[0] != 0 && Settings.ETH_IP[3] != 255;
```

Comment:
This I don't get.
The last byte of an IP-address can be 255. It all depends on the subnet mask whether or not it is a broadcast address.

Reply:
Yes that's right. I will change it.

Reply:
But how to change it?
It is the same way than für wifi check of static IPs in ESPEasyWifi.cpp:
```
bool useStaticIP() {
  return Settings.IP[0] != 0 && Settings.IP[0] != 255;
}

```

Shoud I change both to
` return Settings.IP[0] != 0;`
?

Reply:
You check the **first** byte against 0 and the **last** byte against 255.
The existing check tests the **first** byte.

Reply:
Oh yes, sorry. I need som ne glasses.
I chaged it to
`bool ethUseStaticIP() {
  return Settings.ETH_IP[0] != 0 && Settings.ETH_IP[0] != 255;
}`